### PR TITLE
Created documentation page for generating manifest files from search api, and a link to that page from index.md

### PR DESCRIPTION
**Pull request by DerekFurstPitt**

### CODE REVIEW

File: docs/clt/generate-manifest.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+page where you can find a breakdown of its different endpoints as well as example queries. 
+
+The following endpoints support generating a manifest file:
+
+* ```/search```
+* ```/<index_without_prefix>/search```
+* ```/param-search/<entity_type>```
+
+To geneate a manifest, append the argument `?produce-clt-manifest=true` to the URL. So for ```/search``` for example, using the base SearchAPI URL
+"https://search.api.hubmapconsortium.org", we get "https://search.api.hubmapconsortium.org/search?produce-clt-manifest=true".
```
Comment (yuanzhou) [inline docs/clt/generate-manifest.md]: Use `https://search.api.hubmapconsortium.org/v3/` as base URL

File: docs/clt/generate-manifest.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+Adding this argument prevents the endpoint from returning its normal result. Only the manifest will be furnished. Additionally, 
+the manifest will be generated without paths specified to individual resources within a given dataset. The entire directory will be included as
+denoted by the "/" in the manifest. 
+
+#### Example query and Manifest
+
+The following is an example of a query,the URL given to generate a manifest, and the manifest to be generated:
+
+https://search.api.org/hubmapconsortium.org/search?produce-clt-manifest=true"
```
Comment (yuanzhou) [inline docs/clt/generate-manifest.md]: Use `https://search.api.hubmapconsortium.org/v3/` as base URL

File: docs/clt/generate-manifest.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        {
+          "match_phrase": {
+            "donor.group_name": "Vanderbilt TMC"
+          }
+        }
+      ],
+      "filter": [
+        {
+          "match": {
+            "entity_type.keyword": "Sample"
```
Comment (yuanzhou) [inline docs/clt/generate-manifest.md]: I don't think Sample would result any manifest... Use Dataset instead.

**Review by yuanzhou — CHANGES_REQUESTED** [review]

**Review by shirey — CHANGES_REQUESTED** [review]
@DerekFurstPitt Did you make changes to the openapi "*spec.yaml" file to include instructions for ?produce-clt-manifest=true?

Also, please add some content to generate-manifest.md that explains how/has an example of saving the manifest file and using it with the hubmap-clt.  Also, also along with this link to the clt documentation.

In the clt/index.md file-- make sure to tell folks that they can construct manifest files manually too- people are going to follow this VERY explicitly and think that this is the only way to generate manifest files.  So some thing like "Manifest files can be constructed manually as explained below or automatically using a SearchAPI query by specifying an optional argument.  More information . . ."

**Comment by DerekFurstPitt** [comment]
@shirey I made some tweaks per your requests. Please take a look and let me know if any further changes are requested

**Review by shirey — APPROVED** [review]
Looks good

**Review by yuanzhou — APPROVED** [review]
Will hold off the merge until the actual features are released to PROD.

**Pull request merged**